packages/terra-dropdown-button/CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed incorrect role for upcoming axe-core version bump
```

Comment:
nitpick  - can we specify which role and component piece we updated, and why?

Reply:
Updated here: https://github.com/cerner/terra-core/pull/3323/commits/48e71424fd2542e32a670ae28265982b565f3eac